cmd/registry/cmd/compute/lint/lint_test.go

```diff
@@ -30,7 +38,112 @@ func TestMain(m *testing.M) {
 
 func TestLint(t *testing.T) {
 	command := Command()
+	command.SilenceErrors = true
+	command.SilenceUsage = true
 	if err := command.Execute(); err == nil {
 		t.Fatalf("Execute() with no args succeeded and should have failed")
 	}
 }
+
+func TestInvalidComputeLint(t *testing.T) {
+	tests := []struct {
+		desc string
+		args []string
+	}{
+		{
+			desc: "no-linter-specified",
+			args: []string{"spec"},
+		},
+		{
+			desc: "missing-linter-specified",
+			args: []string{"spec", "--linter", "nonexistent"},
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.desc, func(t *testing.T) {
+			command := Command()
+			command.SilenceErrors = true
+			command.SilenceUsage = true
+			command.SetArgs(test.args)
+			if err := command.Execute(); err == nil {
+				t.Fatalf("Execute() with no args succeeded and should have failed")
+			}
+		})
+	}
+}
+
+func TestComputeLint(t *testing.T) {
+	project := names.Project{ProjectID: "lint-test"}
+	ctx := context.Background()
+	registryClient, _ := grpctest.SetupRegistry(ctx, t, project.ProjectID, nil)
+
+	config, err := connection.ActiveConfig()
+	if err != nil {
+		t.Fatalf("Setup: Failed to get registry configuration: %s", err)
+	}
+	config.Project = project.ProjectID
+	connection.SetConfig(config)
+
+	applyCmd := apply.Command()
+	applyCmd.SetArgs([]string{"-f", "../complexity/testdata/apigeeregistry", "-R"})
```

Comment:
Rather than copy this directory here, I am just picking it out of its nearby location, expecting to follow up with a PR that just moves this test data around (with no other changes, to make it easier to review).